tests/cxx/subfolder/indirectSub.h

```diff
@@ -0,0 +1,17 @@
+//===--- direct_near.h - test input file for iwyu -------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+
+#ifndef DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_TESTS_INDIRECT_SUB_H_
+#define DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_TESTS_INDIRECT_SUB_H_
+
+class IndirectSubClass {};
```

Comment:
This name is a little unclear, I read it as sub-class in the "derived-class" sense at first.